### ISSUE TITLE
Support PyCharm 2023.2 EAP

### ISSUE THREAD

**Issue by jerzyk** [trinhanhngoc/pycharm-odoo#382]
```
java.lang.Throwable: Assertion failed: Do not use PsiElement for popup model. See PsiTargetNavigator
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:462)
	at com.intellij.ui.popup.PopupFactoryImpl.createPopupChooserBuilder(PopupFactoryImpl.java:83)
	at com.intellij.codeInsight.daemon.impl.PsiElementListNavigator$NavigateOrPopupHelper.navigateOrCreatePopup(PsiElementListNavigator.java:217)
	at com.intellij.codeInsight.daemon.impl.PsiElementListNavigator.navigateOrCreatePopup(PsiElementListNavigator.java:135)
	at com.intellij.codeInsight.daemon.impl.PsiElementListNavigator.navigateOrCreatePopup(PsiElementListNavigator.java:111)
	at com.intellij.codeInsight.daemon.impl.PsiElementListNavigator.openTargets(PsiElementListNavigator.java:56)
	at com.intellij.codeInsight.daemon.impl.PsiElementListNavigator.openTargets(PsiElementListNavigator.java:47)
	at dev.ngocta.pycharm.odoo.model.OdooModelLineMarkerProvider.lambda$static$0(OdooModelLineMarkerProvider.java:50)
	at com.intellij.codeInsight.daemon.NavigateAction.actionPerformed(NavigateAction.java:61)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.java:333)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lambda$performActionDumbAwareWithCallbacks$4(ActionUtil.java:307)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareWithCallbacks(ActionUtil.java:356)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.java:307)
	at com.intellij.openapi.editor.impl.EditorGutterComponentImpl.performAction(EditorGutterComponentImpl.java:2291)
	at com.intellij.openapi.editor.impl.EditorGutterComponentImpl.invokeGutterAction(EditorGutterComponentImpl.java:2233)
	at com.intellij.openapi.editor.impl.EditorGutterComponentImpl.mouseReleased(EditorGutterComponentImpl.java:2211)
	at com.intellij.openapi.editor.impl.EditorImpl.processMouseReleased(EditorImpl.java:2378)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseAdapter.lambda$runMouseReleasedCommand$1(EditorImpl.java:4008)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:219)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:174)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseAdapter.runMouseReleasedCommand(EditorImpl.java:4010)
	at com.intellij.openapi.editor.impl.EditorImpl$MyMouseAdapter.mouseReleased(EditorImpl.java:3895)
	at java.desktop/java.awt.AWTEventMulticaster.mouseReleased(AWTEventMulticaster.java:298)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6657)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
	at java.desktop/java.awt.Component.processEvent(Component.java:6422)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4954)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4581)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4522)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2808)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:791)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:740)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:734)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:764)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:762)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:685)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:633)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:588)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:791)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:992)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:992)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:869)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

PyCharm 2023.2 EAP (Professional Edition)
Build #PY-232.6734.4, built on May 31, 2023
dev.ngocta.pycharm-odoo (2022.7.2)

**Comment by trinhanhngoc** [comment]
Hi @jerzyk , You are using an old version of the Odoo plugin (2022.7.2). The latest version is 2023.6.0, which doesn't yet support PyCharm 2023.2 EAP.

**Comment by jerzyk** [comment]
so this is an issue with latest EAP - it does not upgrade plugin to newest version

**Comment by jerzyk** [comment]
it is showing 2022.7.2 as a latest version

<img width="692" alt="Screenshot 2023-06-07 at 13 40 40" src="https://github.com/odoo-ide/pycharm-odoo/assets/199073/9b0e50b5-c576-4845-82f7-a773a9dd64d3">

**Comment by jerzyk** [comment]
when trying to install newest version from disk:

<img width="452" alt="Screenshot 2023-06-07 at 13 46 10" src="https://github.com/odoo-ide/pycharm-odoo/assets/199073/1371ce6c-d58c-48a9-bce6-25f1cae1e886">

**Comment by trinhanhngoc** [comment]
> it is showing 2022.7.2 as a latest version
> 
> <img alt="Screenshot 2023-06-07 at 13 40 40" width="692" src="https://user-images.githubusercontent.com/199073/244043895-9b0e50b5-c576-4845-82f7-a773a9dd64d3.png">

Yes. This is a feature. PyCharm will show the latest version compatible with the version of PyCharm in use. I am working on supporting PyCharm 2023.2 EAP. Please wait for a new update tomorrow.

**Comment by jerzyk** [comment]
Thanks!

**Comment by trinhanhngoc** [comment]
Hi @jerzyk The new version 2023.6.0.232 has been released for PyCharm 2023.2.
